purchasables.md

```diff
@@ -1,14 +1,132 @@
 # Purchasables
 
-A purchasable is a custom element type that can be sold through the cart. Commerce includes [product variant](products.md#variants) and [donation](donations.md) purchasables, but [plugins and modules](https://docs.craftcms.com/v3/extend) can provide their own.
+The core of anything sold with Commerce is a purchasable, a custom [element type](https://docs.craftcms.com/v3/extend/element-types.html) that can be added to a cart.
+
+It should be extended to add useful attributes like Commerce’s included [product variant](products.md#variants) and [donation](donations.md) purchasables. [Plugins and modules](https://docs.craftcms.com/v3/extend) can provide their own purchasables.
+
+## Purchasables and Line Items
+
+Every purchasable is destined to become a line item, which happens when the purchasable is added to a cart. This is an important shift where the purchasable goes from describing an item’s content and available options to a line item describing part of a financial transaction.
+
```

Comment:
something more like "describing an item’s content and available options to describing a line item as it exists in the order"

Reply:
I used the term *financial transaction* to try and emphasize the product going from a concept to a cost. I guess maybe it got weird.

Reply:
Yeah financial transaction sounds like a payment transaction concept which is unreleated.

Reply:
thanks mate